src/movieList/components/Filters/filters.css

```diff
@@ -0,0 +1,61 @@
+.container__filter {
+    display: flex;
+    justify-content: space-between;
+}
+
+.block__sort {
+    display: flex;
+    margin-right: 40px;
+    justify-content: space-between;
+    width: 300px;
+    align-items: center;
+}
+
+.container__filter {
+    margin-top: 5px;
+}
+
+.btn__filter {
+    background-color: #4e4e4e;
+    color: aliceblue;
+    font-size: 13px;
+    padding: 10px;
+    border-radius: 5px;
+    cursor: pointer;
+}
+
+.icon-sort {
+    height: 20px;
+    width: 20px;
+}
+
+.btn__filter-complete {
+    background: rgba(255, 164, 164, 0.626) linear-gradient(rgba(255, 255, 255, 0.7), rgba(255, 255, 255, 0.474));
+    color: rgb(255, 0, 93);
+    font-size: 13px;
+    padding: 10px;
+    border-radius: 5px;
+    cursor: pointer;
```

Comment:
`.btn__filter-complete` should only add/update values to `.btn__filter`. Not copy the same values from it.
```
font-size: 13px;
padding: 10px;
border-radius: 5px;
cursor: pointer;
```
shouldn't be there